tsl/test/expected/continuous_aggs_policy.out

```diff
@@ -0,0 +1,264 @@
+DELETE FROM _timescaledb_config.bgw_job WHERE TRUE;
+SET ROLE :ROLE_DEFAULT_PERM_USER;
+SELECT count(*) FROM _timescaledb_config.bgw_job;
+ count 
+-------
+     0
+(1 row)
+
+\set ON_ERROR_STOP 0
+SELECT add_refresh_continuous_aggregate_policy('int_tab', '1 day'::interval, 10 , '1 h'::interval); 
+ERROR:  "int_tab" is not a continuous aggregate
+SELECT add_refresh_continuous_aggregate_policy('mat_m1', '1 day'::interval, 10 , '1 h'::interval); 
+ERROR:  invalid parameter value for start_interval
+SELECT add_refresh_continuous_aggregate_policy('mat_m1', '1 day'::interval, 10 ); 
+ERROR:  function add_refresh_continuous_aggregate_policy(unknown, interval, integer) does not exist at character 8
+SELECT add_refresh_continuous_aggregate_policy('mat_m1', 10, '1 day'::interval, '1 h'::interval); 
+ERROR:  invalid parameter value for end_interval
+SELECT add_refresh_continuous_aggregate_policy('mat_m1', 20, 10, '1h'::interval) as job_id \gset
+--adding again should warn/error 
+SELECT add_refresh_continuous_aggregate_policy('mat_m1', 20, 10, '1h'::interval, if_not_exists=>false); 
+ERROR:  refresh policy already exists for continuous aggregate "mat_m1"
+SELECT add_refresh_continuous_aggregate_policy('mat_m1', 10, 20, '1h'::interval, if_not_exists=>true); 
+WARNING:  could not add refresh policy due to existing policy on continuous aggregate with different arguments
+ add_refresh_continuous_aggregate_policy 
+-----------------------------------------
+                                      -1
+(1 row)
+
+SELECT add_refresh_continuous_aggregate_policy('mat_m1', 20, 10, '1h'::interval, if_not_exists=>true); 
+NOTICE:  refresh policy already exists on continuous aggregate "mat_m1", skipping
+ add_refresh_continuous_aggregate_policy 
+-----------------------------------------
+                                      -1
+(1 row)
+
+-- modify config and try to add, should error out
+SELECT config FROM _timescaledb_config.bgw_job where id = :job_id;
+                               config                               
+--------------------------------------------------------------------
+ {"end_interval": 10, "start_interval": 20, "mat_hypertable_id": 2}
+(1 row)
+
+SELECT hypertable_id as mat_id FROM _timescaledb_config.bgw_job where id = :job_id \gset
+\c :TEST_DBNAME :ROLE_SUPERUSER
+UPDATE _timescaledb_config.bgw_job 
+SET config = jsonb_build_object('mat_hypertable_id', :mat_id)
+WHERE id = :job_id;
+SET ROLE :ROLE_DEFAULT_PERM_USER;
+SELECT config FROM _timescaledb_config.bgw_job where id = :job_id;
+          config          
+--------------------------
+ {"mat_hypertable_id": 2}
+(1 row)
+
+SELECT add_refresh_continuous_aggregate_policy('mat_m1', 20, 10, '1h'::interval, if_not_exists=>true);
+ERROR:  could not find start_interval in config for existing job
+SELECT remove_refresh_continuous_aggregate_policy('int_tab');
+ERROR:  "int_tab" is not a continuous aggregate
+SELECT remove_refresh_continuous_aggregate_policy('mat_m1');
```

Comment:
Nit: Perhaps we should add a NOTICE here acknowledging that the policy was indeed dropped? (probably also applies to all policies).

Reply:
this should be done across policies. out of scope for this PR.